atuin-client/src/history.rs

```diff
@@ -93,7 +93,7 @@ pub struct HistoryStats {
 
 impl History {
     #[allow(clippy::too_many_arguments)]
-    fn new(
+    pub fn new(
```

Comment:
could we keep this private please? History should be created using the builder pattern, hence this is not exposed. there should be a few examples elsewhere in this file

Reply:
Oh i see.  Yep, fixed!